packages/x-components/src/wiring/__tests__/wires-operators.spec.ts

```diff
@@ -129,188 +126,18 @@ describe('testing wires operators', () => {
     });
   });
 
-  describe('testing timing operators', () => {
-    beforeAll(jest.useFakeTimers);
-    afterAll(jest.useRealTimers);
+  describe('testing operator ' + mapWire.name, () => {
+    test(mapWire.name + ' emits the valued transformed by the function passed by parameter', () => {
```

Comment:
Why we are using test instead of it?

Reply:
We should use `test` when we can't write a valid sentence with the description, but as [Jest says](https://jestjs.io/docs/api#testname-fn-timeout), they are both exactly the same, it is just for readability.